snapcraft/__init__.py

```diff
@@ -288,8 +289,8 @@ def env(self, root):
         """
         return []
 
-    def set_target_machine(self):
-        """Set the target compilation architecture to machine."""
```

Comment:
El 11/04/16 a las 14:27, Kyle Fazzari escribió:
> In snapcraft/__init__.py
> <https://github.com/ubuntu-core/snapcraft/pull/434#discussion_r59245215>:
>
> > @@ -288,8 +289,8 @@ def env(self, root):
> >          """
> >          return []
> >  
> > -    def set_target_machine(self):
> > -        """Set the target compilation architecture to machine."""
>
> Can we leave this function, continue taking the |machine| parameter,
> deprecate it, and simply have it call |enable_cross_compilation()|?
While not hard, the target-arch thing was marked API unstable, so not really compelled to do it. Counter thoughts?